gmso/external/convert_hoomd.py

```diff
@@ -1507,11 +1510,11 @@ def _parse_periodic_improper(
 def _validate_base_units(base_units, top, auto_scale, potential_types=None):
     """Validate the provided base units, infer units (based on top's positions and masses) if none is provided."""
     if base_units and auto_scale:
-        warnings.warn(
+        logger.info(
```

Comment:
We might consider making this one a warning. Really, I wonder if this should raise an error, but at the least it would be good to keepthe warning. Also, this isn't a warning that will come up tons of times.

Reply:
Changed to error in 970afa9